core/src/main/java/io/nixer/nixerplugin/core/detection/rules/ratio/FailedLoginRatioRule.java

```diff
@@ -11,6 +11,26 @@
 import org.apache.commons.logging.LogFactory;
 import org.springframework.util.Assert;
 
+/**
```

Comment:
This description is too detailed, should be on the method level. It does not describe minimumSampleSize and at this level of detail, it should describe all logic. 
I would not document in detail these as one can just read the code.

Reply:
Ok, I left here just a brief description.